include/linux/kfence.h

```diff
@@ -69,6 +71,7 @@ static inline bool kfence_discard_slab(struct kmem_cache *s, struct page *page)
 static inline bool kfence_shutdown_cache(struct kmem_cache *s) { return true; }
 static inline void *kfence_alloc(struct kmem_cache *s, size_t size, gfp_t flags) { return NULL; }
 static inline size_t kfence_ksize(const void *addr) { return 0; }
+static inline void *kfence_obj_start(const void *addr) { return NULL; }
```

Comment:
I think for the main APIs like this, we shouldn't abbreviate.

I could offer: `kfence_object_start()`, `kfence_object_of()`, or `kfence_start_of()`.

I was trying to find symmetry / consistency with some existing names, and it reminds me of `container_of()`.

Reply:
I think kfence_object_start() is more descriptive.